Feat(TU-8691): Add preventOpenOnClose option (#639)
Typeform/embed: docs/configuration.md

You can open modal embed types (all except widget) based on user actions.

Properties `open` and `openValue` define when to automatically open the typeform.

> Also, you can define if you wish to prevent automatically re-opening the typeform by setting `preventReOpenOnClose`.

Reviewer: ```suggestion
If you wish to prevent automatically re-opening the typeform you can set `preventReOpenOnClose`.
```
